Gemfile

```diff
@@ -36,6 +36,9 @@ gem "dotenv", ">= 2.4.0", "< 3.0.0"
 # Caching for octokit operations
 gem "faraday-http-cache", ">= 2.0.0", "< 3.0.0"
 
+# Add foreman for `bundle exec rake <prod, dev>` command
+gem "foreman", ">= 0.82.0"
```

Comment:
There is a warning on https://github.com/ddollar/foreman:
`Ruby users should take care not to install foreman in their project's Gemfile.`
🤔